build.gradle.kts

```diff
@@ -114,29 +117,66 @@ subprojects {
             html.enabled(true)
             xml.enabled(true)
         }
-
-        tasks.withType<Javadoc>().all { enabled = false }
     }
 
     if (!isSample) {
         apply {
-            plugin(BintrayRelease.plugin)
-            plugin(Ktlint.plugin)
-        }
+            plugin("org.gradle.maven-publish")
```

Comment:
in the long form necessary ?
would ´ plugin("maven-publish")´ not work?

Reply:
Odd, you have `maven-publish` in the main plugins block

Reply:
I need 2 places or it will not be able to sync. Do you have a clue why though?